.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
```

Comment:
We should probably remove the linting from `kwalitee` anyway and extract commit message check to separate package.